src/app/services/baw-api/website-status/website-status.service.ts

```diff
@@ -0,0 +1,27 @@
+import { HttpClient } from "@angular/common/http";
+import { Inject, Injectable } from "@angular/core";
+import { ApiShow } from "@baw-api/api-common";
+import { WebsiteStatus } from "@models/WebsiteStatus";
+import { API_ROOT } from "@services/config/config.tokens";
+import { Observable, catchError, map, of } from "rxjs";
+
+@Injectable()
+export class WebsiteStatusService implements ApiShow<WebsiteStatus> {
+  public constructor(
+    private http: HttpClient,
+    @Inject(API_ROOT) private apiRoot: string
+  ) {}
+
+  // if the server is fully down, we will not receive a response from the API
+  // we can therefore not determine the status of the server and we should return "null"
+  public show(): Observable<WebsiteStatus | null> {
+    const endpoint = `${this.apiRoot}/status`;
+
+    return this.http
+      .get<WebsiteStatus>(endpoint)
```

Comment:
does this send the `accept: application/json` header?

Reply:
No it didn't

Fixed

![image](https://github.com/QutEcoacoustics/workbench-client/assets/33742269/d0bd56bd-2ef6-4f0c-8c2d-fdee090af0ec)